Mathlib/Algebra/Order/Floor.lean

```diff
@@ -8,11 +8,10 @@ import Mathlib.Data.Int.Lemmas
 import Mathlib.Data.Int.CharZero
 import Mathlib.Data.Set.Intervals.Group
 import Mathlib.Data.Set.Lattice
-import Mathlib.Init.Meta.WellFoundedTactics
+import Mathlib.Init.Data.Nat.Lemmas
 import Mathlib.Tactic.Abel
 import Mathlib.Tactic.Linarith
 import Mathlib.Tactic.Positivity
-import Mathlib.Init.Data.Nat.Lemmas
```

Comment:
You have no idea how many edits these lists went through, some things were removed and then undone later because the removal didn't work, or other things like that. That said I did try to manually reformat the import lists where reasonable, and I think this one was manually reformatted to put `Tactic` lines at the end and `Init` lines at the beginning. It would be nice to have some style guide for this kind of thing.

Reply:
There seems to be a tendency to put imports for newly added files in the same order as those imports appear in `Mathlib.lean`.